.rubocop.yml

```diff
@@ -6,6 +6,7 @@ AllCops:
     - 'lib/graphql/language/parser.rb'
     - 'gemfiles/**/*'
     - 'tmp/**/*'
+    - 'vendor/**/*'
```

Comment:
For some reason if I don't add this my postgresql build will fail on thousands of lint issues in the vendor folder.